charts/integration/templates/integration-integration.yaml

```diff
@@ -83,7 +83,7 @@ spec:
   restartPolicy: Never
 
   initContainers:
-  # FUTUREWORK: reproduce brig's initContainers (for geoip)
+  # FUTUREWORK: reproduce brig's initContainers (despite geoip removed?)
```

Comment:
```suggestion
```

i think leaving this in causes more confusion than it is worth?  your call, though.